releasenotes/notes/wheel-version-status-page-5d15e8241bbf20e1.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Display wheel version in the agent status page (for python checks).
```

Comment:
I'd rephrase to something like "Display the version for Python checks on the status page"